src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -101,7 +117,7 @@ namespace winrt::TerminalApp::implementation
 
             if (const auto selectedItem = _filteredActionsView().SelectedItem())
             {
-                if (const auto data = selectedItem.try_as<Command>())
+                if (const auto data = selectedItem.try_as<TerminalApp::Command>())
```

Comment:
I had to specifically add the `TerminalApp::` part here to disambiguate it from its implementation type, otherwise the compiler would complain. Though I think it's because I included `Command.h` in order to create implementation typed `Command` objects for the TabSwitcher.